src/core/shape_inference/include/avg_pool_shape_inference.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/util.hpp>
```

Comment:
Can we use only new headers?
Also please use "" instead of <> for OV headers

Reply:
Yes, this old util header is not required here.